dashboard/app/assets/javascripts/application.js.erb

```diff
@@ -26,6 +26,7 @@
 //= require md5
 
 // Application assets
+//= require namespace
```

Comment:
Yep!  Good call on adding a comment.  Do you know of a better way to do this so `window.dashboard` is always declared first?